r/R/expression.R

```diff
@@ -30,6 +30,7 @@
   "str_length" = "utf8_length",
   "str_to_lower" = "utf8_lower",
   "str_to_upper" = "utf8_upper",
+  "str_reverse" = "utf8_reverse"
```

Comment:
comma got lost in the merge
```suggestion
  "str_reverse" = "utf8_reverse",
```